cmd/cdi-operator/operator.go

```diff
@@ -98,6 +99,7 @@ func main() {
 		LeaderElectionNamespace:    namespace,
 		LeaderElectionID:           "cdi-operator-leader-election-helper",
 		LeaderElectionResourceLock: "leases",
+		HealthProbeBindAddress:     ":8444",
```

Comment:
not a problem just curious, what's the issue with the default?

Reply:
ah okay my bad. newly created operator-sdk projects set this to 8081 on the cli flag level, which apparently is problematic https://github.com/openshift/dpu-operator/commit/acc0afd41ea524790a214114fd7bec922e16a304